tests/components/unifiprotect/test_repairs.py

```diff
@@ -0,0 +1,156 @@
+"""Test repairs for unifiprotect."""
+
+from __future__ import annotations
+
+from copy import copy
+from http import HTTPStatus
+from unittest.mock import ANY, Mock
+
+from pyunifiprotect.data import Version
+
+from homeassistant.components.repairs.issue_handler import (
+    async_process_repairs_platforms,
+)
+from homeassistant.components.unifiprotect.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from .utils import MockUFPFixture, init_entry
+
+
+async def test_ea_warning_ignore(
+    hass: HomeAssistant,
+    ufp: MockUFPFixture,
+    hass_client,
+    hass_ws_client,
+):
+    """Test EA warning is created if using prerelease version of Protect."""
+
+    version = ufp.api.bootstrap.nvr.version
+    assert version.is_prerelease
+    await init_entry(hass, ufp, [])
+    await async_process_repairs_platforms(hass)
+    ws_client = await hass_ws_client(hass)
+    client = await hass_client()
+
+    await ws_client.send_json({"id": 1, "type": "repairs/list_issues"})
+    msg = await ws_client.receive_json()
+
+    assert msg["success"]
+    assert len(msg["result"]["issues"]) == 1
+    issue = msg["result"]["issues"][0]
+    assert issue["issue_id"] == "ea_warning"
+
+    url = "/api/repairs/issues/fix"
```

Comment:
It looks like this is hard coded on the frontend
```
src/data/repairs.ts:  hass.callApi<DataEntryFlowStep>("POST", "repairs/issues/fix", {
src/data/repairs.ts:  hass.callApi<DataEntryFlowStep>("GET", `repairs/issues/fix/${flowId}`);
src/data/repairs.ts:  hass.callApi<DataEntryFlowStep>("POST", `repairs/issues/fix/${flowId}`, data);
src/data/repairs.ts:  hass.callApi("DELETE", `repairs/issues/fix/${flowId}`);
```

Maybe make the url a constant in case it ever changes?